README.md

```diff
@@ -1,2 +1,26 @@
 # ci-ansible-molecule-container
 Provides a Docker image that contains ansible and molecule and will be used to provision and run tests
+
+## Prerequisites
+
+The scripts have been developed and tested using:
+
+- [Docker](https://www.docker.com/) (2.3.0.3)
+
+## Software
+
+The container has been built with the following versions installed on it:
+
+Name                    | Description                                                          | Example
```

Comment:
I'd change this to just have columns: `Name` and `Version`, with values like `Ansible` and `2.9.10` respectively. Otherwise it's like we're describing versions rather than software as it were